src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/executablestatementcount/InputExecutableStatementCountLambdas.java

```diff
@@ -0,0 +1,34 @@
+package com.puppycrawl.tools.checkstyle.checks.sizes.executablestatementcount;
+
+/* Config:
+ * max = 1
+ * tokens = LAMBDA
+ */
+
+import java.util.function.Consumer;
```

Comment:
minor: please place comments before `Config` comment section

Reply:
is this what was expected?

Reply:
no, I meant that imports should appear before config comment

Reply:
oh, okay. done now.